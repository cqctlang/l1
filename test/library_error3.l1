

evalk("@with_imports(a()*b) { ; }",
    @lambda(r) {r;},
    @lambda() {});

evalk("@with_imports(ab.b.c) { ; }",
    @lambda(r) {r;},
    @lambda() {});

evalk("@with_imports( (f, ab.b.c) ) { ; }",
    @lambda(r) {r;},
    @lambda() {});

evalk("@with_imports( (f.a, ab.b) ) { ; }",
    @lambda(r) {r;},
    @lambda() {});

@library(test) {};

evalk("@with_imports( (f, test.f) ) { ; }",
    @lambda(r) {r;},
    @lambda() {});

evalk("@with_imports( (t, test) ) { t.t(); }",
    @lambda(r) {r;},
    @lambda() {});

