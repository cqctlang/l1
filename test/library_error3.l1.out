error: <eval-input>:1: Bad import specification
error: <eval-input>:1: Bad import specification
error: <eval-input>:1: Bad import specification
error: <eval-input>:1: Bad import specification
error: <eval-input>:1: Attempt to import non-exported symbol f from test
error: attempt to call non-procedure
	#0                   entry	(<eval-input>:1)
	#1                   entry	(library_error3.l1:27)
